src/MoBi.Presentation/Presenter/ChartPresenter.cs

```diff
@@ -54,9 +54,10 @@ public interface IChartPresenter : IPresenter<IChartView>,
       /// <remarks>
       ///    This method ensures the correct order of parameter setting
       /// </remarks>
-      void Show(ICurveChart chart, IReadOnlyList<DataRepository> dataRepositories, CurveChartTemplate defaultTemplate = null);
+      void Show(CurveChart chart, IReadOnlyList<DataRepository> dataRepositories, CurveChartTemplate defaultTemplate = null);
 
       void UpdateTemplatesBasedOn(IWithChartTemplates withChartTemplates);
+      void Edit(CurveChart curveChart);
    }
 
    public abstract class ChartPresenter : AbstractCommandCollectorPresenter<IChartView, IChartPresenter>, IChartPresenter
```

Comment:
Probably that guy should actually inherit from the OSPSuite.Core ChartPresenter no?